v21.2/deploy-app-netlify.md

```diff
@@ -0,0 +1,227 @@
+~~~
+
+The files in the `public` and `src` directories define the frontend for the app and are outside the scope of this tutorial. In the following sections, we focus on connecting to CockroachDB and deploying to Netlify.
+
+## Step 3. Initialize the database
+
+At the top level of the project directory, use the [`cockroach sql` command](cockroach-sql.html) to initialize the `leaderboard` database in your {{ site.data.products.serverless }} cluster:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach sql --url "<connection-string>" --file ./database/schema.sql
+~~~
+
+Where `<connection-string>` is the connection string you obtained earlier from the {{ site.data.products.db }} Console.
+
+This command runs the SQL commands in the `database/schema.sql` file. These commands initialize the database schema in your cluster.
+
+{{site.data.alerts.callout_info}}
+CockroachDB does not support Prisma Migrate. We recommend executing DDL SQL statements to initialize your database schema separately. After the database schema is initialized, you can load existing tables into your Prisma schema with the Prisma Client, as demonstrated [here](https://www.prisma.io/docs/concepts/components/introspection).
+{{site.data.alerts.end}}
+
+## Step 4. Initialize Prisma Client
+
+1. Set the local `DATABASE_URL` environment variable to a valid connection string to the `leaderboard` database on your cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ export DATABASE_URL="postgresql://<username>:<password>@<host>:<port>/leaderboard?sslmode=verify-full&options=--cluster%3D<routing-id>"
+    ~~~
+
+    Where:
+    - `<username>` is the [SQL user](authorization.html#sql-users) connecting to the cluster.
+    - `<password>` is the password for the SQL user connecting to the cluster.
+    - `<host>` is the host on which the CockroachDB node is running.
+    - `<port>` is the port at which the CockroachDB node is listening.
+    - `<routing-id>` is cluster's routing ID (e.g., `funky-skunk-123`). The routing ID identifies your tenant cluster on a [multi-tenant host](../cockroachcloud/architecture.html#architecture).
+
+    {{site.data.alerts.callout_info}}
+    This connection string is identical to the connection string you obtained earlier from the {{ site.data.products.db }} Console, with the exception of the database parameter (`leaderboard`).
+    {{site.data.alerts.end}}
+
+     By default, Prisma uses the local `DATABASE_URL` environment variable as the connection string to the database.
+
+1. Initialize Prisma Client:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ npx prisma generate
+    ~~~
+
+    Your application can now communicate with your CockroachDB cluster using Prisma Client.
+
+## Step 5. Deploy the application
+
+You can deploy web applications directly from GitHub to Netlify. In this tutorial, we use the [Netlify CLI](https://docs.netlify.com/cli) to deploy the app.
```

Comment:
Maybe explicitly call out that users need to install the Netlify CLI.

Reply:
Also this is a 404, should be https://docs.netlify.com/cli/get-started/

Reply:
I think we could even throw in `npm install --global netlify-cli` since it's so simple and doesn't require the user to go to another page.